test/components/helpers/task/checks.coffee

```diff
@@ -87,6 +87,11 @@ checks =
     expect(div.querySelector('.question-feedback').innerHTML).to.equal(feedback_html)
     {div, component, stepId, taskId, state, router, history, correct_answer, feedback_html}
 
+  _checkRecoveryRefreshChoice: ({div, component, stepId, taskId, state, router, history}) ->
+    expect(div.querySelector('.footer-buttons').children.length).to.equal(3)
+    titles = _.pluck(div.querySelector('.footer-buttons').children, 'textContent')
+    expect(titles).to.deep.equal(['Try Another', 'Refresh My Memory', 'Move On'])
```

Comment:
Can you test on a class on the buttons instead of text content? It could be a `-` prefixed class if there's no styling in the LESS files yet